.github/workflows/create-plan-pr.yml

```
@@ -0,0 +1,138 @@
+        run: |
+          if [ -f "branch_name.txt" ]; then
+            BRANCH_NAME=$(cat branch_name.txt)
+            echo "BRANCH_NAME=$BRANCH_NAME" >> $GITHUB_OUTPUT
+            echo "Found branch: $BRANCH_NAME"
+          else
+            echo "Branch name file not found, using fallback"
+            echo "BRANCH_NAME=merge/plan-import-fallback" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create Pull Request
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)